### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by melaniedrevet**
Issue #16

### CODE REVIEW

**Comment by VickiLanger** [comment]
Thank you so much for your work. I'm looking for another person to review before merging. Once they've reviewed, we shall merge your pr.

After that, I'll add a link to this version.

File: site/checklist_fr.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+## Liste de contrôle anti-gardien
+
+La liste de contrôle est divisée en sections pour nous aider à ouvrir les portes et à éviter le contrôle de ces portes.
+
+**Aller à:**
+// A FAIRE: les liens seront bientôt ajoutés
+
+### Inclusion
+ - [ ] Les images doivent varier en fonction de l'âge, des capacités physiques et mentales, du peuple, de la nationnalité, du genre et des types de corps ([ref](https://www.forumone.com/ideas/how-to-choose-diverse-and-inclusive-photos/))
```
Comment (vanakenm) [inline site/checklist_fr.md:18]: Les images doivent présenter des personnes d'âge, de capacité physiques et mentales, d'origine, de genre et de types de corps variés

File: site/checklist_fr.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+ - [ ] Les exemples de noms doivent être différents du status ([ref](https://www.christytuckerlearning.com/diverse-characters-in-learning-scenarios/))
```
Comment (vanakenm) [inline site/checklist_fr.md:19]: Les exemples de noms utilisés doivent sortir du status quo

File: site/checklist_fr.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+ - [ ] Avez-vous utilisé « homme », ou « hommes » ou des mots les contenant pour désigner des personnes qui ne sont peut-être pas des hommes ? ([ref](https://writingcenter.unc.edu/tips-and-tools/gender-inclusive-language/))
+ - [ ] Avez-vous utilisé « il », « lui », « son » ou « lui-même » pour désigner des personnes qui ne sont peut-être pas des hommes ? ([ref](https://writingcenter.unc.edu/tips-and-tools/gender-inclusive-language/))
+ - [ ] Si vous avez mentionné le sexe ou le genre de quelqu'un, était-il nécessaire de le faire ? ([ref](https://writingcenter.unc.edu/tips-and-tools/gender-inclusive-language/))
+ - [ ] Utilisez-vous des stéréotypes professionnels (ou autres) ? ([ref](https://writingcenter.unc.edu/tips-and-tools/gender-inclusive-language/))
+ - [ ] Fournissez-vous les mêmes types d'informations et de descriptions lorsque vous écrivez sur des personnes de genres différents ?([ref](https://writingcenter.unc.edu/tips-and-tools/gender-inclusive-language/))
+ - [ ] Utilisez-vous « ils » comme une option neutre en termes de genre ? ([ref](https://writingcenter.unc.edu/tips-and-tools/gender-inclusive-language/))
```
Comment (vanakenm) [inline site/checklist_fr.md:33]: Utilisez-vous « iel » comme une option neutre en termes de genre ? "Iel" is the equivalent of "they", as: a gender neutral pronoun. Note that this is much less commonly used in French (yet?) than in English, but it just entered "Le Robert", a major French dictionnary.

File: site/checklist_fr.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+    - [ ] pas ~~"Indien donneur"~~ → s'abstenir de l'utiliser ([ref](https://www.ictinc.ca/blog/indian-giver-come-can-give-back))
+    - [ ] pas ~~"Heure indienne", "Été indien"~~ → s'abstenir de l'utiliser ([ref](https://www.ictinc.ca/blog/culturally-offensive-phrases-you-should-use-at))
+    - [ ] pas ~~"Gitan / Juif"~~ → **arnaqué**, **agrippé** ([ref](http://meloukhia.net/2011/02/language_matters_no_gypped_is_pas_a_good_alternative_to_jewed/))
+    - [ ] pas ~~"Gitan"~~ → s'abstenir de l'utiliser ([ref](https://now.org/blog/the-g-word-isnt-for-you-how-gypsy-erases-romani-women/))
+    - [ ] pas ~~"Karma"~~ → **châtiment** ([ref bientôt disponible](A FAIRE))
+
+#### Inclusion du Handicap
+- [ ] Éviter l'inspiration porno ([ref](https://www.youtube.com/watch?v=SxrS7-I_sMQ))
+- [ ] Lorsque vous discutez d'un produit, indiquez ce qui le rend accessible ([ref bientôt disponible](A FAIRE))
+- [ ] Éviter le langage capacitiste ([ref](http://web.augsburg.edu/english/writinglab/Avoiding_Ableist_Language.pdf)), ([ref](https://www.unr.edu/nevada-today/blogs/2020/avoiding-ableist-terminology))
```
Comment (vanakenm) [inline site/checklist_fr.md:55]: I think that the equivalent of "ableism" in french is "Validisme" - against, not yet as common.

"Eviter le language capacististe/le validisme

File: site/checklist_fr.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+    - [ ] pas ~~normal~~ → **commun**, **typique**, **fréquent** ([ref](https://medium.com/@leoji/avoid-the-word-normal-and-use-more-words-709fb6a9a11f))
+    - [ ] pas ~~Fou, dingo, insensé~~ → **bizarre**, **enorme**, **inconnu** ([ref](https://www.selfdefined.app/definitions/crazy/))
+    - [ ] pas ~~Vérification sanitaire~~ → **contrôle confidentiel**, **contrôle de la température** ([ref](https://www.selfdefined.app/definitions/sanity-check/))
+    - [ ] pas ~~Crétin, idiot~~ → À déterminer ([ref](https://www.selfdefined.app/definitions/idiot/))
+    - [ ] pas ~~Bête, stupide~~ → **déraisonnable**, **étrange** ([ref](https://www.selfdefined.app/definitions/stupid/))
+    - [ ] pas ~~Ridicule~~ → **ridicule**, **insensé**, **idiot** ([ref](https://www.selfdefined.app/definitions/derpy/))
```
Comment (vanakenm) [inline site/checklist_fr.md:61]: This one has no french equivalent - I would remove it here

File: site/checklist_fr.md [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+    - [ ] pas ~~Retard*, *tard~~ →  s'abstenir de l'utiliser ([ref](https://www.selfdefined.app/definitions/r-word/))
+    - [ ] pas ~~Boiteux, estropié~~ → **désactivé** ([ref](https://www.selfdefined.app/lame/))
```
Comment (vanakenm) [inline site/checklist_fr.md:63]: handicappé

File: site/checklist_fr.md [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+    - [ ] pas ~~Sourd~~ → **insensible**, **Dans le mauvais goût** ([ref](https://www.selfdefined.app/definitions/tone-deaf/))
+
+#### Inclusion de la Race
+- [ ] Assurer la majuscule de "Noir" ([ref](https://apnews.com/article/entertainment-cultures-race-and-ethnicity-us-news-ap-top-news-7e36c00c5af0436abc09e051261fff1f))
+- [ ] Éviter l'appropriation raciale ([ref bientôt disponible](A FAIRE))
+    - [ ] S'abstenir du visage noir numérique ([ref](https://www.selfdefined.app/definitions/digital-blackface/))
+- [ ] Évitez les termes et expressions racistes
+    - [ ] pas ~~Grand-père~~ → **héritage**, **historique** ([ref](https://www.selfdefined.app/definitions/grandfathering/))
+    - [ ] pas ~~liste noire / liste blanche~~ → **bloquer/autoriser**, **nier/faire confiance**, **exclure/inclure** ([ref](https://www.selfdefined.app/definitions/blacklist-whitelist/))
+    - [ ] pas ~~Maître d'esclave~~ →  **primaire/réplique**, **tronc/branche**, **parent/enfant** ([ref](https://www.selfdefined.app/definitions/master-slave/))
```
Comment (vanakenm) [inline site/checklist_fr.md:79]: Maître/Esclave

File: site/checklist_fr.md [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+- [ ] Évitez de supposer des connaissances digitales approfondies ([ref bientôt disponible](A FAIRE))
+- [ ] Éviter le langage âgiste ([ref bientôt disponible](A FAIRE))
+    - [ ] pas ~~Si facile, ma grand-mère pourrait le faire~~ → s'abstenir de l'utiliser ([ref bientôt disponible](A FAIRE))
+    - [ ] pas ~~Intitulé [insérer la génération]~~ → s'abstenir de l'utiliser ([ref bientôt disponible](A FAIRE))
+
+
+
+### Faites en sorte que les gens se sentent les bienvenus
+- [ ] Avertir avant discussion/images de sujets déclencheurs/traumatiques ([ref bientôt disponible](A FAIRE))
+- [ ] Évitez d'utiliser des mots qui rendent les choses faciles (par exemple, simple, juste, direct, etc.). Au lieu de cela, qualifiez la difficulté : « Cela peut être facile si vous connaissez déjà [cette chose] ». ([ref bientôt disponible](A FAIRE))
```
Comment (vanakenm) [inline site/checklist_fr.md:97]: Évitez d'utiliser des mots qui laissent entendre que des choses sont faciles (par exemple, simple, juste, direct, etc.)

**Review by vanakenm — APPROVED** [review]
Hi! Thanks already for this content and the translation. The exercise was much more difficult than I though - great job there. 

There is a lot of nuance involved, and some things do not translate that easily from a language to another. I put proposals where I felt I could propose an alternative, but those are just that - proposals.

Hope it helps.

File: site/checklist_fr.md [hunk 9]
```diff
@@ -0,0 +1,149 @@
+---
+layout: page
+title: OpenGates Checklist
+permalink: /checklist/fr
+---
+
+
+Une liste de contrôle de référence pour vous aider à éviter le contrôle d'accès dans vos publications, articles, vidéos, discussions, présentations, podcasts, sites et dans tout autre contenu de développement.
```
[the remark this answers is not in the document]
Comment (VickiLanger) [reply site/checklist_fr.md:8]: Good call. I agree "barrière à l'entrée" (barrier to entry) fits a lot bette than "contrôle d'accès".

File: site/checklist_fr.md (hunk 3 above)
Comment (melaniedrevet) [reply site/checklist_fr.md]: Indeed, "iel" has just entered the dictionary and we don't use it that much yet.

**Comment by melaniedrevet** [comment]
Hi ! Indeed, there are a lot of words that don't really have an exact translation in French, that's why sometimes I translated "word by word" even if that didn't make too much sense.

In any case, thank you for your suggestions, I changed all that!

**Pull request merged**